sklearn/multioutput.py

```diff
@@ -844,6 +889,10 @@ class labels for each estimator in the chain.
     order_ : list
         The order of labels in the classifier chain.
 
+    chain_method_ : str
```

Comment:
This would be provided when using `_return_response_method_used` in `_get_response_values`.